astropy/coordinates/tests/test_intermediate_transformations.py

```diff
@@ -164,6 +164,31 @@ def test_cirs_to_altaz():
     assert_allclose(cirs.dec, cirs3.dec)
 
 
+def test_cirs_to_hadec():
+    """
+    Check the basic CIRS<->HADec transforms.  More thorough checks implicitly
+    happen in `test_iau_fullstack`
```

Comment:
Is this statement true? I think we would need to add the relevant tests to `test_iau_fullstack`.

Reply:
No, that was just a copy and paste from `test_cirs_to_altaz()`. I will remove it.